src/shared/sd-dns-resolver.c

```diff
@@ -0,0 +1,362 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "dns-resolver-internal.h"
+
+#include "macro.h"
+#include "unaligned.h"
+#include "socket-netlink.h"
+#include "string-table.h"
+#include "string-util.h"
+#include "strv.h"
+
+void sd_dns_resolver_done(sd_dns_resolver *res) {
```

Comment:
can be static, no?

Reply:
I don't think so?

Reply:
is this called from outside of this? (at least not in this commit?)

Reply:
ah, i see now that it is used plenty outside in later commits. ignore me then.